client/dashboard/me/active-subscriptions/active-subscription-expiry.tsx

```diff
@@ -0,0 +1,323 @@
+import { formatCurrency } from '@automattic/number-formatters';
+import { ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { useLocale } from '../../app/locale';
+import { formatDate } from '../../utils/datetime';
+import { SubscriptionBillPeriod } from './constants';
+import {
+	isTemporarySitePurchase,
+	isA4ATemporarySitePurchase,
+	isRecentMonthlyPurchase,
+	getRelativeTimeString,
```

Comment:
Team review: [nit] Consider using the HD component `<TimeSince>` instead of `getRelativeTimeString()`.

Reply:
`TimeSince` only shows custom text for dates in the past and only for dates within the past 7 days. Very few subscription expiration dates fall into that category. Most will read "in 2 years" or "in 5 months" or "3 months ago".